### DISCUSSION TITLE
Manage nested Relations with Manager

### DISCUSSION THREAD

**Discussion by DasPocky** [filamentphp/filament#8734] [Help]
Hi,

I would like to use a nested relation manager. I currently have a table with "Tours", another with "Stops" and another with "Loads". The separation has different reasons. A load is assigned to a stop and the stop is assigned to a tour. Is there any way to manage both the stops (I managed this with a simple relation manager) and the loads from within the tours?

Greeting,
Björn

**Comment by leandrocfe** [comment]
I would use pages instead of modals. The [Filament demo](https://demo.filamentphp.com/shop/orders) has some examples that you can have a look.
